src/main/java/com/napier/sem/Country.java

```diff
@@ -1,20 +1,79 @@
 package com.napier.sem;
 
-public class Country {
-    public String code; //country code
-
-    public String name; //country name
-    public String continent; //continent
-    public String region; //region
-    public double surfaceArea; //surfaceArea of the country
-    public int indepYear; // independence year
-    public long population; // country's population
-    public double lifeExpectancy; // life expectancy of the danger
-    public double gnp; // GNP
-    public double gnpOld; //GNPOld
-    public String localName; //local name
-    public String governmentForm; // government form
-    public String headOfState; //head of state
-    public int capital; //capital
-    public String code2; //code 2
-}
+
```

Comment:
I think inline comments like before would be a lot more readable

Reply:
Andreas told me to have comments above methods, as it makes the code more readable

+ the code with inline comment is completely gone now